### ISSUE TITLE
.yaml trainer config

### ISSUE THREAD

**Issue by wwwnev** [Olivier-tl/MO-DAGAN#16]
Devrait-on inclure plus d'hyper-paramètres dans les fichiers de config yaml? Genre quel optimizer prendre, quelle loss, cb. d'epochs, etc ?

PS. Est ce que c'est le genre de chose pour laquelle je devrais ouvrir une issue ? :P

**Comment by Olivier-tl** [comment]
Sure! J'pense qu'on peut fixer l'optimizer & la loss, mais le learning rate & le nb d'epochs ça serait bon que ça soit dans la config. Comme  #le dataset & model, on pourrait ajouter qqch comme: 

```
trainer:
    lr: 0.001
    epochs: 100
```
 et on ajoute un attribut trainer_config dans la class Config qu'on passe directement à TrainerFactory.create().

**Comment by Olivier-tl** [comment]
Alors tu peux finir ton ClassificationTrainer avec la config hardcodé dans la classe, et on updatera ça plus tard avec cet issue.

**Comment by Olivier-tl** [comment]
Ah j'viens de voir que tu as déjà codé plusieurs optimizers & le code pour choisir une loss! 👌🏻 Let's go with something like that then:
```
trainer:
    lr: 0.001
    epochs: 100
    optimizer: adam
    loss: cross-entropy
```
